sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -1333,40 +1333,33 @@ private void ValidateRecognizedForm(RecognizedForm recognizedForm, bool includeF
                 Assert.That(field.Confidence, Is.GreaterThanOrEqualTo(0.0).Within(0.01));
                 Assert.That(field.Confidence, Is.LessThanOrEqualTo(1.0).Within(0.01));
 
-                var labelData = field.LabelData;
-
-                if (labelData != null)
-                {
-                    Assert.Greater(labelData.PageNumber, 0);
-
-                    if (labelData.BoundingBox.Points != null)
-                    {
-                        Assert.AreEqual(4, labelData.BoundingBox.Points.Length);
-                    }
-
-                    Assert.NotNull(labelData.FieldElements);
+                ValidateFieldData(field.LabelData, includeFieldElements);
+                ValidateFieldData(field.ValueData, includeFieldElements);
```

Comment:
`FieldData` validation logic has been moved to a new method since now `LabelData` and `ValueData` behave the same way (which is great!).